docs/servers/lan.md

```diff
@@ -0,0 +1,173 @@
+When complete, it should look like this:
+
+<button onclick="document.getElementById('gameinfo-image').classList.toggle('shown')" class='gameinfoButton'>Example Image</button>
+
+<img id="gameinfo-image" class='hidden' src="/images/gameinfo.png">
+
+## KZ Plugins
+
+### Installing the KZ Plugin
+
+1. Open [cs2kz-metamod releases](https://github.com/KZGlobalTeam/cs2kz-metamod/releases) and download the latest ``(pre-) release`` for your operating system.
+
+2. Extract the ``zip`` (Windows) or ``tar.gz`` (Linux), then drag the ``addons`` and ``cfg`` folders into the ``\csgo\`` folder (The same folder used in the [Metamod Setup](#metamod-setup) step).
+
+### Installing Optional Plugins
+
+#### Jumpstat/Quake sounds support
+
+1. Open [MultiAddonManager](https://github.com/Source2ZE/MultiAddonManager/releases) Releases and download the latest release for your operating system (Windows/Linux).
+
+2. Extract the ``zip`` (Windows) or ``tar.gz`` (Linux), then drag the ``addons`` and ``cfg`` folders into the ``\csgo\`` folder.
+
+#### Database Support
+
+1. Open [SQLMM's Releases](https://github.com/zer0k-z/sql_mm/releases) and download the latest release for your operating system (Windows/Linux).
+
+2. Extract the ``zip`` (Windows) or ``tar.gz`` (Linux), then drag the ``addons`` and ``cfg`` folders into the ``\csgo\`` folder.
+
+#### Language Support
+
+1. Open [GetClientCvarValue's](https://github.com/komashchenko/ClientCvarValue/releases) Releases and download the latest release for your operating system (Windows/Linux).
+
+2. Extract the ``zip`` (Windows), then drag the ``addons`` and ``cfg`` folders into the ``\csgo\`` folder.
+
+## Maps
+
+1. Open up [Steam Workshop](https://steamcommunity.com/workshop/browse/?appid=730&searchtext=kz_) and look for a map or collection of maps
+
+2. Edit the ``start.bat`` (created in [Server Setup](#server-setup)) and add either ``+host_workshop_map`` or ``+host_workshop_collection`` command along with the
+``ID`` of the map/collection to the 2nd row on the file. If you've followed this guide exactly as instructed it should look something like this:
+
+```AddingMap
+cd ".\steamcmd\steamapps\common\Counter-Strike Global Offensive\game\bin\win64\"
+start cs2.exe -dedicated +map de_dust2 +sv_setsteamaccount GSLT_TOKEN +host_workshop_map MAP_ID
+```
+
+or
+
+```AddingCollection
+cd ".\steamcmd\steamapps\common\Counter-Strike Global Offensive\game\bin\win64\"
+start cs2.exe -dedicated +map de_dust2 +sv_setsteamaccount GSLT_TOKEN +host_workshop_collection COLLECTION_ID
+```
+
+Replacing ``MAP_ID``/``COLLECTION_ID`` with the correct ID & ``GSLT_TOKEN`` with your token.
+
+> [!INFO]
+> Although maps can be manually added into files, ``FastDL`` does not exist for CS2, so using workshop instead is recommended.
+
+> [!WARNING]
+> Currently the CS2 Workshop doesn't require you to use a GSLT token for maps, but this could change in the future.
```

Comment:
```suggestion
> [!WARNING]
> Currently the Steam Workshop doesn't require you to use an api key for CS2 maps, but this could change in the future.
```